src/three-domain-secure/component.jsx

```diff
@@ -76,8 +76,10 @@ export function getThreeDomainSecureComponent(): TDSComponent {
       props: {
         action: {
           type: "string",
-          queryParam: true,
-          value: () => "verify",
+          // $FlowFixMe[incompatible-call]
+          queryParam: ({ value }) => (value ? "action" : true),
```

Comment:
do we need this? this seems like it is naming the query param `action` in either case. it feels like we could just do `queryParam: true`

Reply:
Good question, this stuff has faded from my memory but I remember we were fiddling with this a bit. @siddy2181 do you recall why we defined it this way?